codegen/cpp/fletchgen/CMakeLists.txt

```
@@ -42,6 +42,24 @@ if(NOT TARGET fletcher::c)
   add_subdirectory(../../../common/c common-c)
 endif()
 
+include(CMakeRC.cmake)
```

Instead of adding the CMakeRC.cmake file to our repository we should include it with `FetchContent`. The copied file is also missing the license information.

```suggestion

FetchContent_Declare(cmakerc
  GIT_REPOSITORY  https://github.com/vector-of-bool/cmrc.git
  GIT_TAG         master
)
FetchContent_MakeAvailable(cmakerc)

```

---

What I did is exactly what they themselves recommend you do in their readme (and I'm aware that this doesn't include their license; it's weird, but hey, they're the ones who told me to do it this way). I've personally come to avoid FetchContent where alternatives are feasible due to its network dependence and speed (especially compared to just including a file directly), but whatever.